qdrant-landing/content/documentation/Tutorials & Examples/cloud-inference-hybrid-search.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Using Cloud Inference with Qdrant for Vector Search
+weight: 35
+---
+# Using Cloud Inference with Qdrant for Vector Search
+## Install Qdrant Client
+```bash
+pip install qdrant-client
```

Comment:
HF datasets are used to load the data, but the library is not installed as a dependency.